src/components/Carousel/Carousel.js

```diff
@@ -6,16 +6,22 @@ import './Carousel.css'
 import Hero from '../Hero/Hero.js'
 
 const Carousel = ({movies}) => {
-  const featuredFilms = movies.slice(10, 15).map(movie => {
-    return (
-      <Hero 
-        title={movie.title} 
-        image={movie.backdrop_path} 
-        key={movie.id}
-        id={movie.id}
-      />
-    )
-  })
+  let featuredFilms = [];
+
+  if (movies.length) {
+    featuredFilms = movies.slice(10, 15).map(movie => {
+      return (
+        <Hero 
+          title={movie.title} 
+          image={movie.backdrop_path} 
+          key={movie.id}
+          id={movie.id}
+        />
+      )
+    })
+  } else {
+    featuredFilms.push(<div className='hero__placeholder' key='hero-placeholder'></div>);
+  }
```

Comment:
I have 2 things here when reading the rubric it mentioned that we shouldn't mutate props passed down. 

1. Do you think that slice method counts as a mutation? 
2. After looking at this Im wondering if the if else statement is needed. We could have the featureFilm array and another declaration 
`const placeHolder = <div className='hero__placeholder' key='hero-placeholder'></div>`
then conditionally render further down the line `{!movies.length && {placeHolder}}`
Its same same but different, just thought I would note it.

Reply:
Also When does it make sense to conditionally render in the main app as rather than a child. I HAVE SO MANY QUESTIONS

Reply:
I don't think the `slice` counts as a mutation since it doesn't mutate the original array

Reply:
2. YES LET'S DO IT